src/core/mavsdk.h

```diff
@@ -131,17 +131,6 @@ class Mavsdk {
         int remote_port,
         ForwardingOption forwarding_option = ForwardingOption::ForwardingOff);
 
-    /**
-     * @brief Adds a TCP connection with a specific port number on localhost.
-     *
-     * @param remote_port The TCP port to connect to (defaults to 5760).
-     * @param forwarding_option message forwarding option (when multiple interfaces are used).
-     * @return The result of adding the connection.
-     */
-    ConnectionResult add_tcp_connection(
-        int remote_port = DEFAULT_TCP_REMOTE_PORT,
-        ForwardingOption forwarding_option = ForwardingOption::ForwardingOff);
-
```

Comment:
aha, so that one was actually not implemented? :sweat_smile:

Reply:
Looks like it.